### ISSUE TITLE
Active learning command line UI error

### ISSUE THREAD

**Issue by jkatofsky** [plastering/plastering#3]
First off, apologies for the accidental pull request - I meant to do that to my fork of the repo!

Anyways, I've been experimenting with Plaster for a bit now and am running into an error with the command line interface.

In the mongo DB I have saved the raw metadata (and only the raw metadata) for a building with the goal of actively learning its points. The entries for our building look as follows in the database:

`{ "_id" : ObjectId("5d0a7de174bc790fc560c339"), "srcid" : "Building_OAT", "building" : "psu_thomas", "metadata" : { "VendorGivenName" : "Building_OAT" } }`

I am using the Zodiac inferencer for this with the following code:

```
srcids = RawMetadata.objects(building=building).distinct("srcid")
zodiac = ZodiacInterface(target_building=building, target_srcids=srcids)
zodiac.update_model(srcids)
metrics = zodiac.evaluate(srcids)
```

When the code is run, the command line interface prompts me to enter a tagset, and when I do, I recieve this error: 

```
File "zodiac_test.py", line 85, in active_learning_all_points
    zodiac = ZodiacInterface(target_building=building, target_srcids=all_srcids)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/zodiac.py", line 200, in __init__
    self.update_model(seed_srcids)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/zodiac.py", line 326, in update_model
    super(ZodiacInterface, self).update_model(new_srcids)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/inferencer.py", line 197, in update_model
    self.ask_example(srcid)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/inferencer.py", line 137, in ask_example
    self.ui.ask_example(srcid, self.target_building, required_label_types)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/uis/cmdline_ui.py", line 261, in ask_example
    self.store_example(srcid, building, answers)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/uis/cmdline_ui.py", line 264, in store_example
    insert_groundtruth(srcid, building, **answers)
TypeError: insert_groundtruth() missing 1 required positional argument: 'pgid'
```
When I tried to modify the code to give `insert_groundtruth` a `pgid` by defaulting the value to "master", I receive this error:

```
Traceback (most recent call last):
  File "zodiac_test.py", line 102, in <module>
    active_learning_all_points(building)
  File "zodiac_test.py", line 86, in active_learning_all_points
    zodiac = ZodiacInterface(target_building=building, target_srcids=all_srcids)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/zodiac.py", line 200, in __init__
    self.update_model(seed_srcids)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/zodiac.py", line 326, in update_model
    super(ZodiacInterface, self).update_model(new_srcids)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/inferencer.py", line 196, in update_model
    self.ask_example(srcid)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/inferencers/inferencer.py", line 136, in ask_example
    self.ui.ask_example(srcid, self.target_building, required_label_types)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/uis/cmdline_ui.py", line 260, in ask_example
    self.store_example(srcid, building, answers)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/uis/cmdline_ui.py", line 263, in store_example
    insert_groundtruth(srcid, building, "master", **answers)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/plastering/metadata_interface.py", line 81, in insert_groundtruth
    obj.save()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/mongoengine/document.py", line 369, in save
    self.validate(clean=clean)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/mongoengine/base/document.py", line 392, in validate
    raise ValidationError(message, errors=errors)
mongoengine.errors.ValidationError: ValidationError (LabeledMetadata:5d0a98f774bc790fc5625983) (Field is required: ['point_tagset'])
```

`insert_groundtruth` takes in the parameter of a `pgid` as of [this commit](https://github.com/plastering/plastering/commit/b6b04817eb63ebce62943958520930d227367995), but the `cmdline_ui` file has not been modified to pass `insert_groundtruth` a `pgid` to then save to the _labelled_ metadata database.

Thank you in advance for any insight or changes regarding this issue!

**Comment by jbkoh** [comment]
Hey, thanks for the report. As you mentioned, I haven't checked the UI part with the pgid update. Now I have an actual user so should be more careful about maintaining the repo :)

Your problem should have been solved by 83f99d095827895a873e40bb575ba3762980d8b7, but note that the UI has not been fully verified. I will work more on this from July, but let me know if you have any questions.

**Comment by jkatofsky** [comment]
This fixed my issue, thank you!
